app/views/content/fund-your-training/_nav-cards.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/fund-your-training/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+            </p>
+            <span class="icon"></span>
+        </a>
+
+        <a href="/fund-your-training/tuition-fees-and-maintenance-loans" class="category__nav-card">
+            <h2>Tuition fees and maintenance loans</h2>
+
+            <p>
+                Find out how much money you could get to help pay your fees and living expenses while you train.
+            </p>
+            <span class="icon"></span>
+        </a>
+
+        <a href="/fund-your-training/scholarships-and-bursaries" class="category__nav-card">
+            <h2>Scholarships and bursaries</h2>
+
+            <p>
+                There is extra funding available for some subject areas, find out if you are eligible.
+            </p>
+            <span class="icon"></span>
+        </a>
+
+        <a href="/fund-your-training/if-youre-disabled" class="category__nav-card">
+            <h2>Support if you’re disabled</h2>
```

Comment:
The nav cards to me seem like they fit into two distinct sections - tuition fees and maintenance loans with scholarships and bursaries, and then the support if you're disabled, a parent or carer, or a non-UK citizen. Be good to add in Gary's caption elements to this page to split these up.